todocli/task.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from enum import Enum
+
+from todocli.todo_api_util import api_timestamp_to_datetime
+
+
+class Task:
+    class Status(Enum):
+        Completed = "completed"
+        NotStarted = "notStarted"
+        InProgress = "inProgress"
+        WaitingOnOthers = "waitingOnOthers"
+        Deferred = "deferred"
+
+    class Importance(Enum):
+        Low = "low"
+        Normal = "normal"
+        High = "high"
+
+    def __init__(self, query_result_task):
```

Comment:
`query_result_task` is accessed very often below. Lets shorten the variable name for readability. `q_result`, `info`, `task_info` etc. At the very least perhaps `query_result`?

Reply:
Renamed it to query_result